drivers/overlay/ov_serf.go

```diff
@@ -146,6 +146,7 @@ func (d *driver) processQuery(q *serf.Query) {
 		return
 	}
 
+	logrus.Debugf("Sending peer query resp %v", fmt.Sprintf("%s %s %s", peerMac.String(), net.IP(peerIPMask).String(), vtep.String()))
```

Comment:
I am taking the same `fmt.Sprintf` we have in the next line and throwing it in a debug. Are you suggesting I print the fields individually in the debug ?